sdk/keyvault/keyvault-admin/src/lro/selectiveKeyRestore/operation.ts

```diff
@@ -65,32 +59,6 @@ export class KeyVaultSelectiveKeyRestorePollOperation extends KeyVaultAdminPollO
     super(state, { cancelMessage: "Cancelling a selective Key Vault restore is not supported." });
   }
 
-  /**
-   * Tracing the selectiveRestore operation
-   */
-  private selectiveRestore(
-    keyName: string,
-    options: SelectiveKeyRestoreOperationOptionalParams,
-  ): Promise<SelectiveKeyRestoreOperationResponse> {
-    return tracingClient.withSpan(
```

Comment:
I'll restore the tracing operation, just spiking out what the changes would look like for now